src/inspector/models/general/playback/internal_models/glissandoplaybackmodel.cpp

```diff
@@ -54,6 +56,17 @@ void GlissandoPlaybackModel::resetProperties()
     m_styleType->resetToDefault();
 }
 
+bool GlissandoPlaybackModel::isHarpGliss() const
+{
+    for (mu::engraving::EngravingItem* element : selection()->elements()) {
```

Comment:
I think you should use `m_elementList` here instead, and also update this value every time in `requestElements()` (so it can't be a `CONSTANT` `Q_PROPERTY`)